packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -1531,97 +1531,85 @@ class _FlameChartHelpDialog extends StatelessWidget {
 
   Widget _buildNavigationInstructions(ThemeData theme) {
     return Row(
-      children: [
-        SizedBox(
-          width: firstColumnWidth,
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.end,
-            children: [
-              Text(
-                'click + drag • ',
-                style: theme.fixedFontStyle,
-              ),
-              Text(
-                'click + fling • ',
-                style: theme.fixedFontStyle,
-              ),
-              Text(
-                'alt/option + scroll • ',
-                style: theme.fixedFontStyle,
-              ),
-              Text(
-                'scroll left / right • ',
-                style: theme.fixedFontStyle,
-              ),
-              Text(
-                'a / d • ',
-                style: theme.fixedFontStyle,
-              ),
-            ],
+      SizedBox(
+        width: firstColumnWidth,
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.end,
+          Text(
+            'click + drag • ',
+            style: theme.fixedFontStyle,
+          ),
+          Text(
+            'click + fling • ',
+            style: theme.fixedFontStyle,
+          ),
+          Text(
+            'alt/option + scroll • ',
+            style: theme.fixedFontStyle,
+          ),
+          Text(
+            'scroll left / right • ',
+            style: theme.fixedFontStyle,
+          ),
+          Text(
+            'a / d • ',
+            style: theme.fixedFontStyle,
```

Comment:
I miss the extra indent that separates the set of children from the other properties on the Column. Now the children are at the same indent as the widget properties.